build.gradle

```diff
@@ -160,6 +160,10 @@ allprojects {
       url 'https://splunk.jfrog.io/splunk/ext-releases-local'
       content { includeGroupByRegex('com\\.splunk\\..*') }
     }
+    maven {
+      url 'https://gitlab.com/api/v4/projects/19871573/packages/maven'
```

Comment:
is this URL stable?

Reply:
It's [the documented source to use.](https://gitlab.com/gitlab-org/security-products/analyzers/fuzzers/javafuzz#installing), so it it advertised as stable.